# 緊急停止システムの自己診断を要求(L-MESのみ)
---

